scripts/setup.sh

```diff
@@ -53,7 +53,7 @@ if [ ${PWD##*/} != $EXPECTED_PWD ]; then
 fi
 
 export OPPIA_DIR=`pwd`
-export COMMON_DIR=$OPPIA_DIR/..
+export COMMON_DIR=$(cd $OPPIA_DIR/..; pwd)
```

Comment:
I had to modify PYTHONPATH to include GitPython, but relative paths didn't seem to work correctly in PYTHONPATH (ie, Python would not find the modules that were being referenced to). Is it worth a comment?